lib/logstash/filters/mutate.rb

```diff
@@ -349,6 +350,7 @@ def lowercase(event)
     #see comments for #uppercase
     @lowercase.each do |field|
       original = event.get(field)
+      next unless original
```

Comment:
Perhaps use `next if original.nil?` instead so we don't skip non-truthy booleans?